src/fidgets/farcaster/components/Embeds/OnchainEmbed.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const OnchainEmbed = ({ url }: { url: string }) => {
+  return (
+    <div key={`onchain-embed-${url}`} className="text-foreground/80 text-sm">
+      {url}
```

Comment:
What is this embed? Should this be wrapped in a link?

Reply:
It is a `chain://` link that the browser can't resolve

See `src/fidgets/farcaster/components/Embeds/index.tsx:33` for when we render this component